swift-service/src/main/java/com/facebook/swift/service/ThriftServer.java

```diff
@@ -112,7 +117,16 @@ public ThriftServer(NiftyProcessor processor, ThriftServerConfig config)
 
     public ThriftServer(NiftyProcessor processor, ThriftServerConfig config, Timer timer)
     {
-        this(processor, config, timer, DEFAULT_FRAME_CODEC_FACTORIES, DEFAULT_PROTOCOL_FACTORIES, DEFAULT_WORKER_EXECUTORS, DEFAULT_SECURITY_FACTORY, DEFAULT_SSL_SERVER_CONFIGURATION);
+        this(
```

Comment:
The code style is to use a continuation indent of 8, so these should be 8 spaces:
```java
this(
        processor,
        config,
        timer,
```
Or you can put the first one on the same line:
```java
this(processor,
        config,
        timer,